cluster/images/etcd-version-monitor/etcd-version-monitor.go

```diff
@@ -218,7 +219,8 @@ func main() {
 	// Register the metrics we defined above with prometheus.
 	prometheus.MustRegister(etcdVersion)
 	prometheus.MustRegister(etcdGRPCRequestsTotal)
-	prometheus.Unregister(prometheus.NewGoCollector()) // Unregister go metrics.
+	prometheus.Unregister(prometheus.NewGoCollector())                     // Unregister go metrics.
+	prometheus.Unregister(prometheus.NewProcessCollector(os.Getpid(), "")) // Unregister process metrics.
```

Comment:
As I wrote yesterday on IM we need an information when the process was started in order to export cumulative metrics. I'm fine with leaving some extra metrics which contains that information, so basically please remove this line.